x/wasmplus/keeper/querier_test.go

```diff
@@ -36,24 +37,50 @@ func TestQueryInactiveContracts(t *testing.T) {
 	}
 }
 
-func TestQueryIsInactiveContract(t *testing.T) {
+func TestQueryInactiveContract(t *testing.T) {
 	ctx, keepers := CreateTestInput(t, false, AvailableCapabilities)
 	keeper := keepers.WasmKeeper
 
 	example := InstantiateHackatomExampleContract(t, ctx, keepers)
-
+	contractAddr := example.Contract
 	q := Querier(keeper)
-	rq := types.QueryInactiveContractRequest{Address: example.Contract.String()}
-	res, err := q.InactiveContract(sdk.WrapSDKContext(ctx), &rq)
-	require.NoError(t, err)
-	require.False(t, res.Inactivated)
 
-	// set inactive
-	err = keeper.deactivateContract(ctx, example.Contract)
-	require.NoError(t, err)
+	specs := map[string]struct {
+		srcQuery       *types.QueryInactiveContractRequest
+		expInactivated bool
+		expErr         error
+	}{
+		"query": {
+			srcQuery:       &types.QueryInactiveContractRequest{Address: contractAddr.String()},
+			expInactivated: true,
+		},
+		"query with unknown address": {
+			srcQuery: &types.QueryInactiveContractRequest{Address: RandomBech32AccountAddress(t)},
+			expErr:   wasmtypes.ErrNotFound,
```

Comment:
I think use "types" is better, because we have already utilized it in the querier_test.